### PR TITLE
Update to 0.2.2

### PR DESCRIPTION
**Pull request by jpellizzari**
Also fixes a bad quote string in the instructions

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -8,19 +8,18 @@ The `wego` command-line interface is currently supported on Mac (x86 and Arm), a
 To install the `wego` CLI, please follow the following steps:
 
 ```console
-curl -L https://github.com/weaveworks/weave-gitops/releases/download/v0.2.1/wego-$(uname)-$(uname -m) -o wego
+curl -L "https://github.com/weaveworks/weave-gitops/releases/download/v0.2.1/wego-$(uname)-$(uname -m)" -o wego
```
Comment (sympatheticmoose) [inline docs/installation.md:11]: should we not update to 0.2.2 here and in the suggested output?

Comment (jpellizzari) [reply docs/installation.md:11]: It looks like the site already takes care of that:

Screenshot from 2021-07-29 06-46-53

Comment (sympatheticmoose) [reply docs/installation.md:11]: magic

**Review by sympatheticmoose — APPROVED** [review]
LGTM

File: docs/installation.md (hunk 1 above)
Comment (sympatheticmoose) [reply docs/installation.md:11]: folks could still come directly to the readme tho, assume that would still reference 0.2.1? Or is there more wizardry i'm not seeing...

Comment (jpellizzari) [reply docs/installation.md:14]: I think I needed to change it there as well. Updated.

**Pull request merged**